src/Transition/OutputRulesFiles/OutputChanges9-3-0-to-9-4-0.md

```diff
@@ -6,3 +6,79 @@ This file documents the structural changes on the output of EnergyPlus that coul
 ### Description
 
 This will eventually become a more structured file, but currently it isn't clear what format is best. As an intermediate solution, and to allow the form to be formed organically, this plain text file is being used. Entries should be clearly delimited.  It isn't expected that there will be but maybe a couple each release at most. Entries should also include some reference back to the repo.  At least a PR number or whatever.
+
+
+### Daylight Factors output in EIO and DFS now supports more than two reference points
+
+[PR#8017](https://github.com/NREL/EnergyPlus/pull/8017) changed the output format in both EIO and DFS (when using `Output:DaylightFactors`)
+to support more than two `Daylighting:ReferencePoint`.
+
+#### EIO
+
+The old format looked like the following, and when no second Reference Point exited, it reported zero for that point. Any additional reference point was not reported.
+
+```
+! <Sky Daylight Factors>, MonthAndDay, Zone Name, Window Name, Daylight Fac: Ref Pt #1, Daylight Fac: Ref Pt #2
+ Clear Sky Daylight Factors,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,1.9074E-002,1.8074E-002
+ Clear Turbid Sky Daylight Factors,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,1.4830E-002,1.4700E-002
+ Intermediate Sky Daylight Factors,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,1.0647E-002,1.0547E-002
+ Overcast Sky Daylight Factors,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,1.4127E-002,1.3127E-002
+```
+
+The new format looks like the following, and specifies the name of the reference point in question:
+
+```
+! <Sky Daylight Factors>, MonthAndDay, Zone Name, Window Name, Reference Point, Daylight Factor
+ Clear Sky,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,ZN_1_DAYLREFPT1,1.9074E-002
+ Clear Sky,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,ZN_1_DAYLREFPT2,1.8074E-002
+ Clear Sky,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,ZN_1_DAYLREFPT3,1.7074E-002
+ Clear Turbid Sky,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,ZN_1_DAYLREFPT1,1.4830E-002
```

Comment:
@jmarrec This is a great improvement, but it is not following the proper eio format (the existing code was not either). All of these lines should start with "Sky Daylight Factors" (matching the string in < > in the header line. So, this needs another column for Sky Type:
```
! <Sky Daylight Factors>, Sky Type, MonthAndDay, Zone Name, Window Name, Reference Point, Daylight Factor
Sky Daylight Factors, Clear Sky,01/21,ZN_1,ZN_1_WALL_NORTH_WINDOW,ZN_1_DAYLREFPT1,1.9074E-002
```
```

Reply:
Thanks @mjwitte. I am not too familiar with the EIO format, so I just followed what was there. 
Should I include the leading space for each record or not? Looking at the other sections in the EIO, it seems to be a mixed bag...

basically should my format string be:

```
" Sky Daylight Factors,{},{},{},{},{},{:.4R}\n"
```

or

```
"Sky Daylight Factors,{},{},{},{},{},{:.4R}\n"
```

Example of EIO:

has space:
```
! <Lights Internal Gains Nominal>,Name,Schedule Name,Zone Name,Zone Floor Area {m2},# Zone Occupants,Lighting Level {W},Lights/Floor Area {W/m2},Lights per person {W/person},Fraction Return Air,Fraction Radiant,Fraction Short Wave,Fraction Convected,Fraction Replaceable,End-Use Category,Nominal Minimum Lighting Level {W},Nominal Maximum Lighting Level {W}
 Lights Internal Gains Nominal, CORE_BOTTOM_LIGHTS,BLDG_LIGHT_SCH,CORE_BOTTOM,983.54,52.9,10582.854,10.760,199.921,0.400,0.400,0.200,0.000,1.000,General,0.000,10582.854
```

doesn't have a leading space:

```
! <Zone/Window Adjacency Daylighting Counts>, Zone Name, Number of Exterior Windows, Number of Exterior Windows in Adjacent Zones
Zone/Window Adjacency Daylighting Counts, WEST ZONE,2,-1
```

Reply:
Addressed in https://github.com/NREL/EnergyPlus/pull/8017/commits/0f1f4270b68f93f02340ce45cbf9ad144e4ec51d, with a leading space (which seems to be slightly more common in the EIO)

Reply:
Either one is fine. It's been mixed for a long time. The eio format description in Output Details and Examples does not mention this.